cmd/vfkit/main.go

```diff
@@ -154,6 +156,13 @@ func runVirtualMachine(vmConfig *config.VirtualMachine) error {
 	log.Infof("waiting for VM to stop")
 	for {
 		err := waitForVMState(vm, vz.VirtualMachineStateStopped)
+		if err == vmStateTimeoutErr {
+			continue
+		}
+		if err != nil && errors.Is(err, vmStateTimeoutErr) {
```

Comment:
should this be `if err != nil && !error.Is(err, vmStateTimeoutErr)`?

Reply:
ouch yes, thanks! Use of `errors.Is` was a last minute change, too rushed ^^

Reply:
I changed the whole block doing the error checks, but I can get back to something similar to this if that's better.